### ISSUE TITLE
Error on Build

### ISSUE THREAD

**Issue by Bowiemb** [giTorto/template-openRefine-extension.g8#1]
`src/main/g8/build.xml:3: Element type "project" must be followed by either attribute specifications, ">" or "/>".`

Any advice? I tried hardcoding in the name, but that didn't seem to help. 

I'm trying to learn how to build an extension. I'm currently following your guide, but I'm trying to look at some complete examples like this. 

Thank you!
My best,
Michael

**Comment by Bowiemb** [comment]
Update: I figured out I needed to install sbt and build with sbt. Also, I needed to update build.properties to `sbt.version=1.0.3`
Though, I'm still trying to debug this error: 
`plugins.scala:2:26: not found: type Build
[error] object PluginDef extends Build {`

My best,
Michael

**Comment by giTorto** [comment]
Sorry for the late reply. I have been very busy lately. Are you using https://github.com/foundweekends/giter8/wiki/giter8-templates to build the template?

Best regards,
Giuliano

**Comment by Bowiemb** [comment]
Of course. No problem. 

Do you mean `sbt new template-openRefine-extension.g8` ? Yes, I've installed giter8. 

Though, is it correct in assuming that this creates a basic extension infrastructure with a working command class? My issue is learning how to interact with the backend data. I've read through your excellent tutorial. Though, it would be helpful for me to see some basic examples. 

My best,
Michael

**Comment by giTorto** [comment]
No, I mean http://www.foundweekends.org/giter8/usage.html . However the project has been refactored recently so there should be some problem running it.

To be honest, I don't remember exactly, however yes, that was the aim of template. A good complete example is the extension to extract e-mails. It is very simple and the code is clean.

Bests,
Giuliano
